.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,38 @@
+name: pypi publishing
+
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+'
+    branches: master
+
+
+jobs:
+
+  publish_on_pypi:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: checkout latest commit
+        uses: actions/checkout@v2
+
+      - name: setup python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: create source distribution
+        run: |
+            python setup.py sdist
+
+      - name: create wheel distribution
+        run: |
+          python setup.py sdist
```

Comment:
I guess here we wanna do
```
python setup.py bdist_wheel
```

Reply:
You are totally right, thanks a lot!
Fixed in a6f5372402cbd2b448cf4d7cd311e9b43527d596